user_stories.md

```diff
@@ -0,0 +1,42 @@
+As a Maker
```

Comment:
Very nice idea to add this to a separate file, instead of having such a full readme!